ratis-test/src/test/java/org/apache/ratis/grpc/TestLogAppenderWithGrpc.java

```diff
@@ -124,8 +124,5 @@ private void runTestRestartLogAppender(MiniRaftClusterWithGrpc cluster) throws E
         Assert.assertTrue(reply.isSuccess());
       }
     }
-
-    // assert INCONSISTENCY counter should become 1
-    Assert.assertEquals(1L, leaderMetrics.getRegistry().counter(counter).getCount());
```

Comment:
I think we can assert that leaderMetrics.getRegistry().counter(counter).getCount() >= 1?

Reply:
@lokeshj1703  Thanks for review. I have updated.